src/pages/liity/index.md

```diff
@@ -198,3 +200,72 @@ Kuvaa:
+
+
+### 16. Lippuvyöhykkeet
+
+Reittiopas tukee matkalipputietojen esittämistä käyttäjälle monella tavoin. Reittiehdotuksissa voidaan kertoa tarvittavan lipun tyyppi ja oletushinta,
+mikäli GTFS-data sisältää tarvittavat hintasäännöt. Ehdotukset myös näyttävät lippuvyöhykkeen vaihtumisen pitkin matkaketjua. Taustakartalle voidaan piirtää
+vyöhykkeiden aluerajat. Kun käyttäjä napauttaa karttaa, osoitetun paikan lippuvyöhyke on myös mahdollista kertoa avatulla ponnahdusikkunalla.
+
+GTFS-datan fare rules ja fare attributes -spesifikaatiot löytyvät osoitteesta [gtfs.org/reference/static](https://gtfs.org/reference/static).
+Jos datasta löytyy tarvittavat fare tiedot, niitä voi konfiguroida reittioppaaseen vastaamalla seuraaviin kysymyksiin:
+- Näytetäänkö lippuvyöhykkeet reittiehdotuksissa (kohta 1. kuvassa alla)
+- Näytetäänkö tarvittavat liput reittiehdotuksissa (2)
+- Näytetäänkö lipun oletushinta reittiehdotuksissa (3)
+- Onko olemassa Internet-osoite ohjesivuun, josta löytyy tietoa lippujen hinnoista ja ostamisesta (4)
+- Miten GTFS fare -tiedoissa määritellyt lipputyypit esitetään käyttäjälle? Tekninen tunnus voi olla tarpeen muuttaa ihmisten tuntemaan muotoon.
+
+![](./images/liput.png)
+
+Reittiehdotusten lisäksi lippuvyöhyketietoa voidaan esittää kartalla. Sitä varten tarvitaan geoJSON-muotoista dataa WGS84-koordinaatistossa.
+On suositeltavaa käyttää mahdollisimman pientä pistemäärää (alle 1000 pistettä) ja desimaalitarkkuutta (5 desimaalia), jotta palvelu pysyy kevyenä ja nopeana.
+Paras tulos saadaan, kun käytössä on kaksi hiukan erilaista dataversiota:
+
+1. Aluetarkistusdata. Siinä lippuyöhykkeet on kuvattu suljettuina polygon tai multipolygon kohteina, yksi feature per lippuvyöhyke.
+Kukin feature sisältää properties tiedoissa ominaisuuden 'Zone', jonka arvo on vyöhykkeen tunnus, esimerkiksi 'A'
+
+Esimerkki lippuvyöhykkeiden aluedatasta: [hsl-zone-areas-20190508.json](https://raw.githubusercontent.com/HSLdevcom/pelias-api/master/middleware/config/hsl_zone_areas_20190508.json)
+
+2. Vyöhykkeiden visualisointi kartalle. Lippuvyöhykkeiden rajaviivat kuvataan linestring/multilinestring geometrioina ilman kaksinkertaista piirtoa alueiden välille.
+Lisäksi tiedoston pitäisi sisältää kullekin vyöhykkeelle point tai multipoint feature, joissa määritellään vyöhyketunnusten paikat kartalla. Esimerkki:
+
+```json
+
+   {
+      "type": "Feature",
+      "properties": {
+         "Zone": "C"
+        }
+      },
+      "geometry": {
+        "type": "MultiPoint",
+        "coordinates": [
+          [24.63, 60.258],
+          [25.167, 60.275],
+          [25.041, 60.331],
+          [24.791, 60.321]
+        ]
+      }
+    }
+```
+
+Yllä on C-vyöhykkeelle määritelty 4 tunnusta ympäri vyöhykealuetta.
+
+Esimerkki vyöhykedatasta karttapiirtoa varten: [hsl-zone-lines-20190508.geojson](https://raw.githubusercontent.com/HSLdevcom/digitransit-ui/v2/static/assets/geojson/hsl_zone_lines_20190508.geojson)
```

Comment:
Using HSL data as an example here can be misleading as the icons use Gotham font instead of the Roboto used in the waltti zone datas.

Reply:
Changed to Kuopio.